ghost/admin/app/components/settings/members/stripe-settings-form.js

```diff
@@ -259,6 +259,7 @@ export default class StripeSettingsForm extends Component {
         yield this.ajax.delete(url);
         yield this.settings.reload();
 
+        this.args.reset();
```

Comment:
@minimaluminium Might be a good idea to use the same syntax as below that checks for availability of `reset` in args by doing `this.args.reset?.()`